.github/workflows/style.yml

```diff
@@ -39,3 +40,7 @@ jobs:
     - name: Lint with flake8
```

Comment:
Just a question: is the idea that `pylint` will _replace_ `flake8`, maybe when its errors have been dealt with? Or do we normally have both on a project?

Reply:
We will run both.